### DISCUSSION TITLE
Precipitation and other weather data

### DISCUSSION THREAD

**Discussion by DovydasBiel** [carla-simulator/carla#4251] [Q&A]
Hello,
I have a question that relates to the use of real-life precipitation and other weather data in order to simulate weather conditions as close as possible to a real-world scenario on Carla. In the Carla documentation, the precipitation (and some other weather conditions) is said to be applied by giving an arbitrary number between 0-100 (where 0 relates to "no rain" and 100 to "heavy rain"). These descriptions are a bit too broad for me, so I was wondering if there is a method/system by which those values could be related to real-life precipitation data (and other weather condition data which is applied by using an arbitrary value between 0-100) which is given in "mm" units?
Thank you in advance!

**Comment by oecker** [comment]
Hello,
I have the same issue. @DovydasBiel have you already found out what the values correspond to in a "mm" measurement?
Thanks!
